Wordle.Domain/LetterScore.cs

```diff
@@ -0,0 +1,8 @@
+namespace Wordle.Domain;
+
+public class LetterScore
+{
+    public int Id { get; set; } 
+    public char Letter { get; set; }
+    public Score Eval { get; set; }
+}
```

Comment:
Does a Letter's score need to be a stateful object? could this be calculated on request, and no state saved?

Reply:
(This is something to think about for future refactoring, and not to be addressed right now)

Reply:
This is a good point! LetterScore, with it's eval is helpful to construct a cell with it's background color. But I could just ultimately return what I need from evaluateGuess. I'll keep that in mind when building to see if I can't refactor.